src/test/java/teammates/test/cases/ui/AdminSearchPageActionTest.java

```diff
@@ -12,8 +12,10 @@ public static void classSetUp() {
     
     @Test
     public void testExecuteAndPostProcess() {
-        // TODO : Test is done as a browser test,
-        // cuz gaeSimulation gives problems when rebuilding the document
+        /*
+	 *	Test is done as a browser test, 
+	 *	because otherwise there are problems when rebuilding the document
+	 */
```

Comment:
Keeping the comment structure and indentation as

```
// Test is done...
// because otherwise...
```

would be better in this case :)